concepts/errors-handling/introduction.md

```diff
@@ -1,3 +1,70 @@
 # Introduction
 
-TODO: add introduction for errors-handling concept
+In Go, errors must be the last value returned in a multiple returns function.
```

Comment:
```suggestion
In Go, errors are by convention the last value returned in a multiple returns function.
```
While I agree that "must" conveys the importance of this, I think it's more accurate to state that it's a convention rather than something that is, for example, enforced by the compiler :)